e2etest/pkg/status/l2status.go

```diff
@@ -34,5 +34,11 @@ func L2ForService(cs client.Client, svc *v1.Service) (*v1beta1.ServiceL2Status,
 	if len(statuses) > 1 {
 		return nil, fmt.Errorf("got more than 1 serviceL2Status object: %d", len(statuses))
 	}
-	return &(statuses[0]), nil
+
+	s := statusList.Items[0]
+	if s.Status.Node == "" {
```

Comment:
nit I would put that inside the if
```
if s:= ; s.Status.Node == { 
}
```

Reply:
I think it's cleaner that way, since we return `&s` later